cpu/esp32/Kconfig

```diff
@@ -31,7 +31,7 @@ config HAS_ESP_BLE_ESP32C3
     bool
```

Comment:
I guess this should now be `HAS_ESP_BLE_ESP32x3` to avoid confusion

Or `HAS_ESP_BLE_ESP32_BLE5` ?

Reply:
Espressif provides four different binary Bluetooth libraries:
- `esp32-bt-lib` for ESP32
- `esp32c2-bt-lib` for ESP32-C2
- `esp32c3-bt-lib` for ESP32-C3 and ESP32-S3
- `esp32h2-bt-lib` for ESP32-H2

That is, the binary library used for ESP32-C3 and ESP32-S3 is called `esp32c3-bt-lib`. `Calling the feature `esp_ble_esp32x3` or `esp_ble_esp32_ble5` wouldn't be unique and confusing as well.

What we could do is to have the features `esp_ble_esp32c3` and `esp_ble_esp32s3`. But then we would have to remove the `FEATURES_REQUIRED += `esp_ble_esp32c3` from `pkg/esp32_sdk_lib_bt_esp32c3/Makefile.dep`. In `pkg/esp32_sdk_lib_bt_esp32c3/Kconfig` we could use
```
depends on HAS_ESP_BLE_ESP32C3  || HAS_ESP_BLE_ESP32S3
```

Reply:
Another possibility would be to use an additional package `pkg/esp32_sdk_lib_bt_esp32s3` for ESP32-S3 which pulls the same library `esp32c3-bt-lib` as the ESP32-C3 implementation does.